WooCommerce/Classes/ViewRelated/Orders/Order Creation/CustomerSection/CreateOrderAddressFormViewModel.swift

```diff
@@ -36,6 +37,15 @@ final class CreateOrderAddressFormViewModel: AddressFormViewModel, AddressFormVi
         }
     }
 
+    /// Creates and presents a discard changes action sheet.
+    ///
+    func presentDiscardChangesActionSheet(viewController: UIViewController) {
+        UIAlertController.presentDiscardChangesActionSheet(viewController: viewController) { [weak self] in
+            viewController.dismiss(animated: true, completion: nil)
+            self?.userDidCancelFlow()
+        }
+    }
```

Comment:
Seeing the `ViewModel` having to `import UIKit` raised my alarm 😆 

I think we should keep the `UIKit` bit constrained to the `DiscardChangesWrapper`, what do you think about just allowing the consumer to provide the `title` and the `action button title`, so then we can call the UIKit action sheet inside the wrapper. something like:

```swift
func discardChangesPrompt(title: String,
                          discardButtonTitle: String,
                          canDismiss: Bool,
                          didDismiss: (() -> Void)? = nil) -> some View {
```

Reply:
Updated in [59c090e](https://github.com/woocommerce/woocommerce-ios/pull/6534/commits/59c090eb868f19e7c4e382adfc6cae2fd99e0f5a). I decided to provide a set of default strings (since I expect that in most cases where we have a discard changes action sheet we'll be ok with those defaults) with the option to override them.